README.md

```diff
@@ -133,25 +153,7 @@ const elements = parse(
     </span>
   </p>
 `,
-  {
```

Comment:
Technically, a variable doesn't need to be created here. However, I am okay with this refactor to improve readability.